site/content/en/docs/tasks/run/rayservice.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Run A RayService"
+linkTitle: "RayServices"
+date: 2025-06-30
+weight: 6
+description: >
+  Run a RayService on Kueue.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [RayService](https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayservice-quick-start.html).
+
+Kueue manages the RayService through the RayCluster created for it. Therefore, RayService needs the label of kueue.x-k8s.io/queue-name: user-queue and this label is propagated to the relevant RayCluster to trigger Kueue's management.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. Make sure you are using Kueue v0.6.0 version or newer and KubeRay v1.3.0 or newer.
+
+2. Check [Administer cluster quotas](/docs/tasks/manage/administer_cluster_quotas) for details on the initial Kueue setup.
+
+3. See [KubeRay Installation](https://raw.githubusercontent.com/ray-project/kuberay/v1.3.0/ray-operator/config/samples/ray-service.sample.yaml) for installation and configuration details of KubeRay.
+
+{{% alert title="Note" color="primary" %}}
+RayService is managed by Kueue through RayCluster, and in order to use RayCluster, prior to v0.8.1, you need to restart Kueue after the installation.
+You can do it by running: `kubectl delete pods -l control-plane=controller-manager -n kueue-system`.
+{{% /alert %}}
```

Comment:
I think it is time to drop this note, because it is not needed for any supported version of Kueue (0.11+). 

However, I'm ok to include it for consistency for now, and decide on this in a separate PR. cc @tenzen-y